src/pybind/mgr/dashboard/controllers/cluster.py

```diff
@@ -18,4 +25,77 @@ def list(self):
     @EndpointDoc("Update the cluster status",
                  parameters={'status': (str, 'Cluster Status')})
     def singleton_set(self, status: str):
-        ClusterModel(status).to_db()
+        ClusterModel(status).to_db()  # -*- coding: utf-8 -*-
```

Comment:
```suggestion
        ClusterModel(status).to_db()
```

Reply:
Oops, missed this while cpoying. Thanks!